### PR TITLE
Homepage

### LINKED ISSUE #373: Change signup button landing page
### Description
When you hit "Signup" on the signup page it should redirect you to the login page

### LINKED ISSUE #414: Create a home page/dashboard
### Description
- Develop a homepage or dashboard that users are brought to after logging in
- Need to discuss what all would be on the page

### PR DESCRIPTION
**Pull request by rileycleavenger**
## Describe your changes
Check Discord for a demo. The landing page has an animated MealDealz logo and then once a user signs in it changes to a recipe card preview thing.

## Issue ticket number and link
closes #414
closes #373

### CODE REVIEW

File: Saucier720-app/src/app/core/services/images/images.service.ts
Comment (samforstot) [inline Saucier720-app/src/app/core/services/images/images.service.ts:1]: Is this for getting icons and stuff from google?

File: Saucier720-app/src/assets/images/dev-profile-pictures/steele-dev-profile.jpeg
Comment (samforstot) [inline Saucier720-app/src/assets/images/dev-profile-pictures/steele-dev-profile.jpeg:1]: LMAO

File: Saucier720-app/src/assets/images/dev-profile-pictures/tyler-dev-profile.jpeg
Comment (samforstot) [inline Saucier720-app/src/assets/images/dev-profile-pictures/tyler-dev-profile.jpeg:1]: Munch

File: src/MealDealz.db
Comment (samforstot) [inline src/MealDealz.db:1]: Hate this guy

**Review by samforstot — APPROVED** [review]
LGTM! This is actually insane. I think you really did an awesome job with this. Very much impressed by each feature on the dash. Very excited to officially add this to develop. Only thing I will say is that this was a huge PR. Do you think that in the future for huge features it may be a good idea to make some sort of feature branch so that each component of this could have its own PR to be viewed? I'm not sure if that will over complicate our workflow. Insane job!

File: Saucier720-app/src/app/core/services/images/images.service.ts
Comment (rileycleavenger) [reply Saucier720-app/src/app/core/services/images/images.service.ts:1]: yes, but its for getting recipes images, but has a queries/day limit so decided not to include it

File: Saucier720-app/src/assets/images/dev-profile-pictures/tyler-dev-profile.jpeg
Comment (rileycleavenger) [reply Saucier720-app/src/assets/images/dev-profile-pictures/tyler-dev-profile.jpeg:1]: @TylerMetz

File: Saucier720-app/src/assets/images/dev-profile-pictures/steele-dev-profile.jpeg
Comment (rileycleavenger) [reply Saucier720-app/src/assets/images/dev-profile-pictures/steele-dev-profile.jpeg:1]: i forgot to update it lol

File: src/MealDealz.db
Comment (rileycleavenger) [reply src/MealDealz.db:1]: facts

File: Saucier720-app/src/assets/images/dev-profile-pictures/steele-dev-profile.jpeg
Comment (steeleelliott03) [reply Saucier720-app/src/assets/images/dev-profile-pictures/steele-dev-profile.jpeg:1]: Woah there 
buddy

**Review by steeleelliott03 — APPROVED** [review]
Lot's of new changes and additions. This is impressive riley, good job.

**Pull request merged**